lib/1.4/dml-builtins.dml

```diff
@@ -997,6 +999,48 @@ template init_as_subobj is (connect, init, _qname) {
     }
 }
 
+/**
+### map\_target
+
+A `connect` object can instantiate the template `map_target`, making it only
+accept objects that are valid map targets, i.e. objects for which
+`SIM_new_map_target` returns a valid pointer, without exception. The template
+provides a session variable `map_target` of type `map_target_t *` which can be
+used to issue transactions to the connected object. The template also defines a
+method `issue(transaction_t *t, uint64 addr) -> (exception_type_t)` which can
+be used as a shorthand for `SIM_issue_transaction`.
+*/
+template map_target is connect {
+    session map_target_t *map_target;
+
+    method validate(conf_object_t *obj) -> (bool) default {
```

Comment:
Having validate/set declared `default` is in principle API. Maybe if it should be documented (not sure). I was going to ask for use cases, but I can see pretty clear use cases myself (like a connect that combines map target with some other interface).